clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -1055,4 +1069,61 @@ private String getAsgName(com.netflix.spinnaker.clouddriver.titus.client.model.J
     }
     return asgName;
   }
+
+  /**
+   * Collect all jobs with the same server group name and different job IDs.
+   *
+   * <p>Log out all the duplicate IDs, and log the cached job ID too.
+   */
+  private void logDuplicateServerGroups(
```

Comment:
🤔  Might simplify this to just mark server groups as seen through the cache process.